meta/documents/changelog_de.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes für Elastic Export Check24.de
 
+## v1.2.3 (2019-09-20)
+
+### Behoben
+Format **Check24DE Fashion**:
+- Wenn es bei der Erstellung oder Aktualisierung einer SKU für eine Variante zu einem Fehler kommt, wird dieser nun gelogged und die Variante trotzdem exportiert.
```

Comment:
"Wenn bei Anlage oder Aktualisierung einer SKU..., wird dieser nun in den Logs aufgezeichnet und die Variante dennoch exportiert."

Reply:
Wenn es beim Anlegen oder Aktualisieren einer SKU für eine Variante zu einem Fehler kommt, wird dieser Fehler nun gelogged und die Variante trotzdem exportiert.